docs/user/04-metrics.md

```diff
@@ -404,3 +404,22 @@ Up to three MetricPipeline resources at a time are supported.
    1. Check which SDK version you are using for instrumentation. 
    1. Investigate whether it is compatible with the OTel collector version.
    1. If required, upgrade to a supported SDK version.
+
+- Symptom: Custom metrics don't arrive at the destination and the OTel Collector produces log entries "Failed to scrape Prometheus endpoint":
+```
+2023-08-29T09:53:07.123Z	warn	internal/transaction.go:111	Failed to scrape Prometheus endpoint	{"kind": "receiver", "name": "prometheus/app-pods", "data_type": "metrics", "scrape_timestamp": 1693302787120, "target_labels": "{__name__=\"up\", instance=\"10.42.0.18:8080\", job=\"app-pods\"}"}
+```
```

Comment:
```suggestion
  ```
```

Reply:
This is displayed as if we're removing the backticks. 
However, that's not what I'm suggesting - we should simply slap 2 blank spaces to the beginning of the line:

<img width="424" alt="image" src="https://github.com/kyma-project/telemetry-manager/assets/76950046/6d7df66c-7c1c-4624-90dd-415fcad7d672">